### PR TITLE
Update Service Mesh Projects

### PR DESCRIPTION
**Pull request by keithmattix**
Update networking projects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ Projects listed on this page are open source that Microsoft maintain or contribu
...
...                                                                     | A standard interface for service meshes on Kubernetes                                                                                                                                                                                                                         | Deprecated; consider using [Gateway API](https://gateway-api.sigs.k8s.io/)                                                                 | [#smi](https://cloud-native.slack.com/messages/smi) <br> [GitHub issues](https://github.com/servicemeshinterface/smi-spec/issues)                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                                               |
```
Comment (ritazh) [inline README.md:34]: Can you pls move this one and all other deprecated stuff to the past project table below?

xref: https://github.com/Azure/container-upstream/pull/142/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53

Comment (keithmattix) [reply README.md:34]: Sorry for the delay! done

Comment (ritazh) [reply README.md:38]: Did you mean to remove this one since it's archived?

Comment (keithmattix) [reply README.md:38]: Ah boo bad rebase

**Review by ritazh — APPROVED** [review]
LGTM

**Pull request merged**